### ISSUE TITLE
Prowl notifications ignored by alarm-notify.sh

### ISSUE THREAD

**Issue by mismacku** [netdata/netdata#6021]
I'll say a big thanks again for netdata! I really appreciate the work.  
  
##### Bug report summary
Prowl notifications work only if one or more other notifications are configured. In my case, if I configure only Prowl notifications, `/usr/libexec/netdata/plugins.d/alarm-notify.sh test "sysadmin"` fails. If I configure Prowl and Slack, I get both Prowl and Slack notifications.
[health_alarm_notify.conf](https://github.com/netdata/netdata/files/3182959/health_alarm_notify.conf.txt)

##### OS / Environment
Baremetal 
Linux 4.15.0-15-generic #16~16.04.6-Ubuntu SMP Thu Apr 5 12:19:23 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux  

##### Netdata version (ouput of `netdata -V`)
netdata v1.14.0-55-nightly  

##### Component Name
health  

##### Steps To Reproduce
Running a notifications test with both Prowl and Slack configured I get.  
`/usr/libexec/netdata/plugins.d/alarm-notify.sh test "sysadmin"`
```
# SENDING TEST WARNING ALARM TO ROLE: sysadmin
2019-05-15 07:12:03: alarm-notify.sh: INFO: sent slack notification for: control test.chart.test_alarm is WARNING to '#monitoring'
2019-05-15 07:12:03: alarm-notify.sh: INFO: sent prowl data for: control test.chart.test_alarm is WARNING
# OK

# SENDING TEST CRITICAL ALARM TO ROLE: sysadmin
2019-05-15 07:12:04: alarm-notify.sh: INFO: sent slack notification for: control test.chart.test_alarm is CRITICAL to '#monitoring'
2019-05-15 07:12:04: alarm-notify.sh: INFO: sent prowl data for: control test.chart.test_alarm is CRITICAL
# OK

# SENDING TEST CLEAR ALARM TO ROLE: sysadmin
2019-05-15 07:12:05: alarm-notify.sh: INFO: sent slack notification for: control test.chart.test_alarm is CLEAR to '#monitoring'
2019-05-15 07:12:05: alarm-notify.sh: INFO: sent prowl data for: control test.chart.test_alarm is CLEAR
# OK
```
Running a notifications test with Prowl only I get.  
`bash -x /usr/libexec/netdata/plugins.d/alarm-notify.sh test "sysadmin"`
[health.log](https://github.com/netdata/netdata/files/3182982/health.log)

Running ` bash -x /usr/libexec/netdata/plugins.d/alarm-notify.sh sysadmin control 1 1 3 1557929846 test_alarm test.chart test.family CLEAR CRITICAL 100 90 /usr/libexec/netdata/plugins.d/alarm-notify.sh 1 3 units 'this is a test alarm to verify notifications work' 'new value' 'old value' 'evaluated expression' 'expression variable values' 0 0`   
I get  health02.log (link removed for security reasons)

##### Expected behavior
I would like to have only Prowl notifications configured for some machines.

**Comment by cakrit** [comment]
Thank you for the excellent report. The fix is in #6022, it will be merged as soon as the checks pass.
FYI, I noticed in the log that you had `role_recipients_prowl[sysadming]`. Perhaps the `g` was added by mistake. 
I will be editing your post to remove the reference to the second log, you don't want people to know your key.

**Comment by mismacku** [comment]
Wow that was quick!  
**sysadming** must have been a vim fat finger.  Thanks for catching that.
I quickly scanned that log, but somehow missed the key. Big thanks for removing it!  

_bows to the monitoring gods_
